sympy/physics/continuum_mechanics/cable.py

```diff
@@ -0,0 +1,334 @@
+        >>> c.supports
+        {'A': [3, 4], 'B': [4, 5]}
+        >>> c.remove_supports()
+        >>> c.supports
+        {}
+        """
+        if len(self._supports) == 0:
+            raise ValueError("There are no supports to remove")
+
+        else:
+            self._supports.clear()
+            self._left_support.clear()
+            self._right_support.clear()
+            self._loads.clear()
+            self._loads_position.clear()
+            self._length = 0
+
+    def change_support(self, label, new_support):
+        """
+        This method changes the mentioned support with a new support.
+
+        Parameters
+        ==========
+        label: String or symbol
+            The label of the support to be changed
+
+        new_support: Tuple of the form (new_label, x, y)
+            new_label: String or symbol
+                The label of the new support
+
+            x: Sympifyable
+                The x-coordinate of the position of the new support.
+
+            y: Sympifyable
+                The y-coordinate of the position of the new support.
+
+        Examples
+        ========
+
+        >>> from sympy.physics.continuum_mechanics.cable import Cable
+        >>> c = Cable()
+        >>> c.add_supports(('A', 3, 4), ('B', 4, 5))
+        >>> c.supports
+        {'A': [3, 4], 'B': [4, 5]}
+        >>> c.change_support('B',('C',5,6))
+        >>> c.supports
+        {'A': [3, 4], 'C': [5, 6]}
+        """
+        if label not in self._supports:
+            raise ValueError("No support exists with the given label")
+
+        else:
+            if label == self._support_labels[0]:
+                if new_support[1] > self._right_support[0]:
+                    raise ValueError("x coordinate of left support should be less than its right support counterpart")
+                else:
+                    self._supports.pop(label)
+                    self._left_support.clear()
+                    self._support_labels.remove(label)
+
+                    x = sympify(new_support[1])
+                    y = sympify(new_support[2])
+                    self._left_support = [x, y]
+                    self._supports[new_support[0]] = [x, y]
+                    self._support_labels.insert(0, new_support[0])
+
+            else:
+                if new_support[1] < self._left_support[0]:
+                    raise ValueError("x coordinate of right support should be greater than its left support counterpart")
+
+                else:
+                    self._supports.pop(label)
+                    self._right_support.clear()
+                    self._support_labels.remove(label)
+
+                    x = sympify(new_support[1])
+                    y = sympify(new_support[2])
+                    self._right_support = [x, y]
+                    self._supports[new_support[0]] = [x, y]
+                    self._support_labels.append(new_support[0])
+
+    def add_loads(self, *args):
```

Comment:
Instead of having `*args`, specify each parameter. See how beam does it
https://github.com/sympy/sympy/blob/9d4517c6b1840aa2c2dfd1617aea32ccedc7041d/sympy/physics/continuum_mechanics/truss.py#L452